src/Server/Remote/NativeSsh.php

```diff
@@ -230,12 +206,55 @@ protected function getConnectionHash()
     protected function getMultiplexingSshOptions()
     {
         return [
-            '-o ControlMaster=auto',
-            '-o ControlPersist=5',
-            '-o ControlPath=\'' . $this->getConnectionHash() . '\'',
+            '-o' => [
+                'ControlMaster=auto',
+                'ControlPersist=5',
+                'ControlPath=' . $this->getConnectionHash(),
+            ],
```

Comment:
I changed structure of options due to new way of building command line arguments.